client/components/wcc-settings-form/settings-group.js

```diff
@@ -56,13 +57,26 @@ const SettingsGroup = ( props ) => {
 			);
 
 		case 'actions':
+			const buttons = group.items.map( ( button ) => {
+				let onClick = noop;
+				let isDisabled = false;
+				let label = button.title;
+				let isPrimary = false;
+				if ( 'submit' === button.type ) {
+					isPrimary = true;
+					if ( form.isSaving ) {
+						isDisabled = true;
+						label = button.progressTitle || label;
```

Comment:
Before we were actually ignoring the `layout` of the `actions` group. We can now add the `Saving...` string to the server-generated existing layouts if we want.